php-classes/TagItem.class.php

```diff
@@ -155,7 +155,7 @@ public static function getTagsSummary($options = array())
                 ,TagItem::getColumnName('ContextClass')
                 ,DB::escape($options['Class']::getStaticRootClass())
                 ,TagItem::getColumnName('ContextID')
-                ,implode(",",$classIDs)
+                ,!empty($classIDs) ? implode(",",$classIDs) : [-1]
```

Comment:
@BillClinton hmm this won't work as written I don't think, because we need this line to return a string. Having the "else" be an array would work if it were _inside_ the `implode` call

Since we're already assured that `$classIDs` is an array and want to return a string here, this is the pattern we usually use for these:

```suggestion
                ,count($classIDs) ? implode(",", $classIDs) : '0'
```